3rd-party/web-stories.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Filter to enable web stories built in open graph data from being output.
+ *
+ * @param bool $enabled If web stories open graph data is enabled.
+ *
+ * @return bool
+ */
+function web_stories_disable_open_graph( $enabled ) {
+	$active_modules = Jetpack::get_active_modules();
+	if ( in_array( 'publicize', $active_modules ) || in_array( 'sharedaddy', $active_modules ) ) {
```

Comment:
This does not take into account the `jetpack_enable_opengraph` filter.

Could do this:

```php
if ( apply_filters( 'jetpack_enable_open_graph', false ) ) {
	add_filter( 'web_stories_enable_open_graph', '__return_false' );
}
```

Reply:
@spacedmonkey I actually meant replacing this entire `in_array` condition with the `apply_filters( 'jetpack_enable_open_graph', false )` check.

Reply:
```php
function jetpack_web_stories_disable_metadata( $enabled ) {
	if ( apply_filters( 'jetpack_enable_open_graph', false ) ) {
		return false;
	}

	return $enabled;
}

add_filter( 'web_stories_enable_open_graph', 'jetpack_web_stories_disable_metadata' );
add_filter( 'web_stories_enable_twitter_metadata', 'jetpack_web_stories_disable_metadata' );
```